src/controllers/wizardfinishpage.js

```diff
@@ -6,20 +6,13 @@ define(["loading"], function (loading) {
         ApiClient.ajax({
             url: ApiClient.getUrl("Startup/Complete"),
             type: "POST"
-        }).then(function () {
-            Dashboard.navigate("dashboard.html");
+        }).then(function() {
             loading.hide();
             window.location.href = "index.html";
         });
     }
 
     return function (view, params) {
         view.querySelector(".btnWizardNext").addEventListener("click", onFinish);
-        view.addEventListener("viewshow", function () {
-            document.querySelector(".skinHeader").classList.add("noHomeButtonHeader");
-        });
-        view.addEventListener("viewhide", function () {
-            document.querySelector(".skinHeader").classList.remove("noHomeButtonHeader");
-        });
```

Comment:
why remove?

Reply:
> 
> 
> also I cannot find `wizardfinishpage.js` anymore, why is it so?

what did you mean not found

Reply:
I wasn't able to see this `wizardfinishpage.js` file in the diff of current PR. it doesn't matter here anyway as it's a merge from master :)